server/src/main/java/org/opensearch/index/fielddata/SingletonSortedNumericUnsignedLongValues.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
+ * Licensed to Elasticsearch under one or more contributor
```

Comment:
do we need to add this license for new file?

Reply:
Ah we certainly don't need it, thank you